federfrei3/address_mover_trucks
# Bookings dashboard basic version

Testing:
- log in as Joe, check My Reservations under avatar
- repeat for Owner
- repeat for Admin
